docs/DeveloperGuide.md

```diff
@@ -6,19 +17,21 @@ title: Developer Guide
 {:toc}
 
```

Comment:
![image](https://github.com/AY2324S2-CS2103T-F10-1/tp/assets/59340097/644722d4-f532-4103-ab05-eba4fbd7caac)
Viewing this file gives this, which I don't think is supposed to show?

Reply:
Should be resolved! Thank you